onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -131,13 +131,15 @@ def create_attention_node(self, mask_index, q_matmul, k_matmul, v_matmul, q_add,
         weight = helper.make_tensor(name=attention_node_name + '_qkv_weight',
                                     data_type=TensorProto.FLOAT,
                                     dims=[self.hidden_size, 3 * self.hidden_size],
-                                    vals=qkv_weight.flatten().tolist())
+                                    vals=bytes(qkv_weight.flatten()),
```

Comment:
What's the benefit on using raw bytes instead of list?

Reply:
I think that it's because save_external_data() requires tensor to have a "raw data" field

Reply:
I see.